easybuild/easyconfigs/b/Bio-EUtilities/Bio-EUtilities-1.76-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,88 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'Bundle'
+
+name = 'Bio-EUtilities'
+version = '1.76'
+
+homepage = 'https://github.com/bioperl/bio-eutilities'
+description = "BioPerl low-level API for retrieving and storing data from NCBI eUtils"
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+builddependencies = [('binutils', '2.32')]
+dependencies = [
+    ('Perl', '5.30.0'),
+    ('BioPerl', '1.7.2'),
+]
+
+exts_defaultclass = 'PerlModule'
+exts_filter = ("perldoc -lm %(ext_name)s ", "")
+
+exts_list = [
+    ('ExtUtils::MakeMaker', '7.44', {
+        'source_tmpl': 'ExtUtils-MakeMaker-7.44.tar.gz',
```

Comment:
use `%(version)s` here rather than hardcoding the version in the name of the source tarball (same below)